docs/CONTRIBUTING.adoc

```diff
@@ -179,3 +179,38 @@ To submit a pull request:
 . Click the _Create pull request_ button.
 
 The pull request will be built in link:https://ci.jenkins.io[ci.jenkins.io].
+
+== Releases
+
+The releases can be generated by using the `release.sh` script at the root of the repository.
+This scripts uses `git`, `gh`, `jq` and `mvn` to generate the release.
+The script also requires Bash version to be greater than 4.
+Make sure you have all those tools on your system.
+
+You can run the script like this:
+
+[source bash]
+----
+bash release.sh
+----
+
+WARNING: to be able to perform a release, you **must** have write permission on the project repository.
```

Comment:
```suggestion
WARNING: To be able to perform a release, you **must** have write permission on the project repository.
```
Small capitalization suggestion since this would be a sentence.